docs/src/tutorial/receiving_messages.md

```diff
@@ -130,7 +134,7 @@ So let's use a helper function for this:
 # };
 fn spawn_and_log_error<F>(fut: F) -> task::JoinHandle<()>
 where
-    F: Future<Output = io::Result<()>> + Send + 'static,
+    F: Future<Output = Result<()>> + Send + 'static,
```

Comment:
Why this change, where does this `Result` type come from?

Reply:
From:

> type Result<T> = std::result::Result<T, Box<dyn std::error::Error + Send + Sync>>;

which was defined previously in the tutorial. The `F` bound on `Future<Output = io::Result<()>>` is a type and does not compile.